primitive/compute/client.go

```diff
@@ -181,18 +181,22 @@ func (c *Client) Hello() error {
 	if err != nil {
 		return err
 	}
+
+        // Get the API version
+        version := helloResponse.GetVersion()
```

Comment:
The formatting looks a bit off here. if you run `gofmt -s -w .` from the root directory it should clean all this up.